compression/const.go

```diff
@@ -0,0 +1,52 @@
+package compression
```

Comment:
- give proper attribution to seaweedfs, or we will be in [big trouble](https://zhuanlan.zhihu.com/p/45653912)
- why this file called `const`?
- actually I personally don't think `IsGzippableFileType` needs 2 return values...